scripts/workspace-conversion/workspace-to-dataset.py

```diff
@@ -266,6 +267,19 @@ def get_workspace_files(wsAttributes):
     return ret
 
 
+def access_url(workspace):
+    p = re.compile("rawls.dsde-(\\w+)")
+    env = p.search(urlRoot).group(1)
+    if env == "prod":
+        terra_url = "https://app.terra.bio"
+    else:
+        terra_url = f"https://bvdp-saturn-{env}.appspot.com"
```

Comment:
```suggestion
    terra_url = "https://app.terra.bio" if env == "prod"
    else f"https://bvdp-saturn-{env}.appspot.com"
```

Reply:
FYI this makes the lines too long, so after `black` reformatting I get
```
    terra_url = (
        "https://app.terra.bio"
        if env == "prod"
        else f"https://bvdp-saturn-{env}.appspot.com"
    )
```
not sure if this is better / worse than the original `if` at this point

Reply:
Yeah, you're right, I'm not sure now.